File: android/src/main/java/com/reactnativechimesdk/data/RosterAttendee.java
package com.reactnativechimesdk.data;

public class RosterAttendee {

  private String attendeeId;
  private String attendeeName;
  private boolean isMuted;

  public RosterAttendee(String attendeeId, String attendeeName) {
    this.attendeeId = attendeeId;
    this.attendeeName = attendeeName;
  }

  public RosterAttendee(String attendeeId, String attendeeName, boolean isMuted) {
    this.attendeeId = attendeeId;
    this.attendeeName = attendeeName;
    this.isMuted = isMuted;
  }

  public String getAttendeeId() {
    return attendeeId;
  }

  public void setAttendeeId(String attendeeId) {
    this.attendeeId = attendeeId;
  }

  public String getAttendeeName() {
    return attendeeName;
  }

  public void setAttendeeName(String attendeeName) {
    this.attendeeName = attendeeName;
  }

  public boolean isMuted() {
    return isMuted;
  }

  public void setMuted(boolean muted) {
    isMuted = muted;
  }
}

File: android/src/main/java/com/reactnativechimesdk/data/RosterAttendee.kt
/*
 * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
 * SPDX-License-Identifier: Apache-2.0
 */

package com.reactnativechimesdk.data

import com.amazonaws.services.chime.sdk.meetings.audiovideo.SignalStrength
import com.amazonaws.services.chime.sdk.meetings.audiovideo.VolumeLevel
import com.amazonaws.services.chime.sdk.meetings.audiovideo.video.VideoTileState

data class RosterAttendee(
  val attendeeId: String,
  val attendeeName: String,
  val volumeLevel: VolumeLevel = VolumeLevel.NotSpeaking,
  val signalStrength: SignalStrength = SignalStrength.High,
  val isActiveSpeaker: Boolean = false,
  var isMuted: Boolean = false
)
